uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -34,6 +34,19 @@ define([
       angular.extend(scope, Mock);
     }));
 
+    describe('on init', function () {
+      var resolvedData = { foo: 'foo', bar: 'bar' };
+
+      beforeEach(function () {
+        initController(resolvedData);
+      });
+
+      it('copies the passed data in the $scope assignment object', function () {
+        expect(scope.assignment.foo).toBeDefined();
```

Comment:
If you deepClone object (as described above), you don't need to care if it was defined.